website/docs/modules/apply_manual_filter.md

```diff
@@ -0,0 +1,57 @@
+---
+title: ApplyManualVariantFilter
+description: Complex SV genotyping
+sidebar_position: 16
+slug: amvf
+---
+
+[WDL source code](https://github.com/broadinstitute/gatk-sv/blob/main/wdl/ApplyManualVariantFilter.wdl)
+
+This module hard-filters variants (dropping records) using [bcftools](https://github.com/samtools/bcftools). While the 
+workflow is general-purpose, we recommend running it with default parameters to eliminate major sources of false 
+positive variants:
+
+1. Deletions called solely by `Wham`.
+2. SVA MEIs called by `Scramble` with the `HIGH_SR_BACKGROUND` flag.
+
+The following diagram illustrates the recommended invocation order:
+
+```mermaid
+
+stateDiagram
+  direction LR
+    
+  classDef inModules stroke-width:0px,fill:#caf0f8,color:#00509d
+  classDef thisModule font-weight:bold,stroke-width:0px,fill:#ff9900,color:white
+  classDef outModules stroke-width:0px,fill:#caf0f8,color:#00509d
+
+  refcv: RefineComplexVariants
+  amvf: ApplyManualVariantFilter
+  svc: SVConcordance
+  refcv --> amvf
+  amvf --> svc
```

Comment:
Still not sure what you mean by this. Recommended order and dependency are sort of the same in my mind.

Reply:
Recommended order seems to me like it should match the order of workflows in the pipeline diagram and the numbered list in Terra. So introducing the dependency of JoinRawCalls on ClusterBatch seems different from just the recommended order. I understand wanting to communicate those relationships somewhere, just seems like it might be confusing, especially if it's labeled "recommended invocation order." And we did stick with strictly recommended order for GatherSampleEvidence/EvidenceQC/TrainGCNV so it may not be straightforward to start calling these dependency diagrams instead

Reply:
It is probably not a big deal but something I think we should standardize/clarify eventually